src/libstd/sys/unix/ext.rs

```diff
@@ -220,6 +224,27 @@ pub mod fs {
             self.as_inner_mut().mode(mode); self
         }
     }
+
+    /// Creates a new symbolic link on the filesystem.
+    ///
+    /// The `dst` path will be a symbolic link pointing to the `src` path.
```

Comment:
Sure, added a brief note to that effect. I was a little hard to summarize the situation without a very verbose explanation, so I just made sure to include the name of the specific privilege required that allows you to find the explanation on StackOverflow.